extensions/sns/src/commands/prepare_canisters/remove_nns_root.rs

```diff
@@ -0,0 +1,29 @@
+//! Code for executing `dfx sns prepare-canisters remove-nns-root`
+use candid::Principal;
+use clap::Parser;
+use dfx_extensions_utils::call_extension_bundled_binary;
+use std::ffi::OsString;
+use std::path::Path;
+
+/// `dfx sns prepare-canisters remove-nns-root` command line arguments.
+#[derive(Parser)]
+pub struct RemoveNnsRootOpts {
+    canister_ids: Vec<Principal>,
+}
+
+/// Executes `dfx sns prepare-canisters remove-nns-root`
+pub fn exec(opts: RemoveNnsRootOpts, dfx_cache_path: &Path) -> anyhow::Result<()> {
+    let mut args = vec![
+        OsString::from("prepare-canisters"),
+        OsString::from("remove-nns-root"),
+    ];
+    args.extend(opts.canister_ids.iter().map(|id| id.to_string().into()));
+
+    call_extension_bundled_binary("sns-cli", &args, dfx_cache_path)
+        .map(|stdout| println!("{}", stdout))
+        .map_err(|error| {
+            println!("{}", error);
+            error
+        })?;
```

Comment:
@smallstepman I think we should reconsider the semantics of `call_extension_bundled_binary`, to make it less cumbersome to use in the common case.  The common case looks like it would be better to inherit stdout, in order to show the output as it happens, rather than buffering until the end, and rather than making the caller display it manually.